src/main/kotlin/com/cognifide/gradle/aem/tooling/tasks/Tail.kt

```diff
@@ -25,13 +25,13 @@ open class Tail : AemDefaultTask() {
         ShutdownHooks.addShutdownHook {
             shouldRunTailing = false
         }
-        logger.lifecycle("Fetching logs every ${FETCH_INTERVAL_IN_MILLIS}ms")
+        logger.lifecycle("Fetching logs every ${Tail.FETCH_INTERVAL_IN_MILLIS}ms")
```

Comment:
I used space somewhere when formatting time / probably in `Formats`

Reply:
OK